alphastats/gui/utils/options.py

```diff
@@ -5,130 +5,133 @@
                     "label": "Color according to",
                 },
             },
-            "function": state.dataset.plot_sampledistribution,
+            "function": dataset.plot_sampledistribution,
         },
         "Intensity Plot": {
             "settings": {
                 "protein_id": {
-                    "options": state.dataset.mat.columns.to_list(),
+                    "options": dataset.mat.columns.to_list(),
                     "label": "ProteinID/ProteinGroup",
                 },
                 "method": {
                     "options": ["violin", "box", "scatter"],
                     "label": "Plot layout",
                 },
                 "group": {
-                    "options": [None] + state.metadata_columns,
+                    "options": [None] + state[StateKeys.METADATA_COLUMNS],
                     "label": "Color according to",
                 },
             },
-            "function": state.dataset.plot_intensity,
+            "function": dataset.plot_intensity,
         },
         "PCA Plot": {
             "settings": {
                 "group": {
-                    "options": [None] + state.metadata_columns,
+                    "options": [None] + state[StateKeys.METADATA_COLUMNS],
                     "label": "Color according to",
                 },
                 "circle": {"label": "Circle"},
             },
-            "function": state.dataset.plot_pca,
+            "function": dataset.plot_pca,
         },
         "UMAP Plot": {
             "settings": {
                 "group": {
-                    "options": [None] + state.metadata_columns,
+                    "options": [None] + state[StateKeys.METADATA_COLUMNS],
                     "label": "Color according to",
                 },
                 "circle": {"label": "Circle"},
             },
-            "function": state.dataset.plot_umap,
+            "function": dataset.plot_umap,
         },
         "t-SNE Plot": {
             "settings": {
                 "group": {
-                    "options": [None] + state.metadata_columns,
+                    "options": [None] + state[StateKeys.METADATA_COLUMNS],
                     "label": "Color according to",
                 },
                 "circle": {"label": "Circle"},
             },
-            "function": state.dataset.plot_tsne,
+            "function": dataset.plot_tsne,
         },
         "Volcano Plot": {
             "between_two_groups": True,
-            "function": state.dataset.plot_volcano,
+            "function": dataset.plot_volcano,
         },
-        "Clustermap": {"function": state.dataset.plot_clustermap},
-        # "Dendrogram": {"function": state.dataset.plot_dendrogram},  # TODO why commented?
+        "Clustermap": {"function": dataset.plot_clustermap},
+        # "Dendrogram": {"function": state[StateKeys.DATASET].plot_dendrogram},  # TODO why commented?
```

Comment:
The dendrogram is rather redundant as it is part of clustermap output, but that's just my guess.